### PR TITLE
chore: Stop bundling automerge WASM

### PR DESCRIPTION
**Pull request by dmaretskyi**
### Motivation / Background

This Pull Request ...

### CODE REVIEW

File: packages/apps/composer-app/index.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
     <meta
       http-equiv="Content-Security-Policy"
       content="
-        connect-src https: file: ipc: socket: wss://*.dxos.org ws://localhost:*;
+        connect-src https: file: ipc: socket: wss://*.dxos.org ws://localhost:* http://localhost*;
```
Comment (wittjosiah) [inline packages/apps/composer-app/index.html:17]:
```suggestion
        connect-src https: file: ipc: socket: wss://*.dxos.org ws://localhost:* http://localhost:*;
```

File: packages/apps/composer-app/shell.html [hunk 2]
```diff
@@ -6,7 +6,7 @@
     <meta
       http-equiv="Content-Security-Policy"
       content="
-        connect-src https: file: ipc: socket: ws://localhost:*;
+        connect-src https: file: ipc: socket: ws://localhost:* http://localhost:3000;
```
Comment (wittjosiah) [inline packages/apps/composer-app/shell.html:9]:
```suggestion
        connect-src https: file: ipc: socket: ws://localhost:* http://localhost:*;
```

File: packages/core/echo/automerge/scripts/compile.mts [hunk 3]
```diff
@@ -93,4 +98,4 @@ await copyFile(
   'dist/lib/browser/automerge_wasm_bg.wasm',
 );
 
-// bust cache
\ No newline at end of file
+// bust cache
```
Comment (wittjosiah) [inline packages/core/echo/automerge/scripts/compile.mts:101]: 👀

Comment (dmaretskyi) [reply packages/core/echo/automerge/scripts/compile.mts:101]: old stuff..

File: packages/core/echo/automerge/scripts/compile.mts [hunk 4]
```diff
@@ -11,23 +11,28 @@ for (const platform of ['node', 'browser'] as const) {
...
-          automerge: 'src/automerge.ts',
-          'automerge/next': 'src/automerge/next.ts',
-          'automerge-repo': 'src/automerge-repo.ts',
-          'automerge-repo-storage-indexeddb': 'src/automerge-repo-storage-indexeddb.ts',
-        },
+            // automerge_wasm_bg: join(
+            //   dirname(await require.resolve('@automerge/automerge-wasm')),
+            //   '../bundler/automerge_wasm_bg.js',
+            // ),
+            // 'automerge-wasm': 'src/automerge-wasm.ts',
```
Comment (jessmartin) [inline packages/core/echo/automerge/scripts/compile.mts:30]: @dmaretskyi leaving these comments in?

**Pull request merged**